packages/integration-tests/src/test-project-setup/create_empty_amplify_project.ts

```diff
@@ -29,5 +35,24 @@ export const createEmptyAmplifyProject = async (
 
   await setupDirAsEsmModule(projectAmplifyDir);
 
+  await execa(
+    packageManagerProps?.executable ?? 'npm',
+    [
+      packageManagerProps?.installCommand ?? 'install',
+      '-D',
+      '@aws-amplify/backend',
+      '@aws-amplify/backend-cli',
+      '@aws-amplify/auth-construct-alpha',
+      '@aws-amplify/backend-deployer',
+      ...(projectDirName === 'data-storage-auth'
+        ? ['uuid', '@types/uuid']
+        : []),
```

Comment:
How much, if any, of this belongs in "create_empty_amplify_project"? I can buy that installing backend and backend-cli should belong here, but not so sure about the other stuff. Perhaps individual tests should install those packages if needed.